--- libelf-0.8.2/lib/Makefile.in.orig	Sat Oct 13 23:35:22 2001
+++ libelf-0.8.2/lib/Makefile.in	Fri Jul 26 01:16:42 2002
@@ -21,7 +21,6 @@
 exec_prefix = @exec_prefix@
 libdir = @libdir@
 includedir = @includedir@
-installdirs = $(libdir) $(includedir) $(includedir)/libelf
 
 CC = @CC@
 LD = @LD@
@@ -164,20 +163,18 @@
 	@$(RM) -r $(picdir)
 	mkdir $(picdir)
 
-install: all installdirs install-data \
+install: all install-data \
 	install-shared-$(DO_SHLIB) install-compat-$(DO_COMPAT)
 
-installdirs: $(top_srcdir)/mkinstalldirs
-	$(SHELL) $(top_srcdir)/mkinstalldirs $(installdirs)
-
 install-data:
-	$(INSTALL_DATA) libelf.a $(libdir)
+	$(INSTALL) -d $(DESTDIR)$(libdir) $(DESTDIR)$(includedir)/libelf
+	$(INSTALL_DATA) libelf.a $(DESTDIR)$(libdir)
 	-cd $(libdir) && $(RANLIB) libelf.a
 	files="$(HDRS) $(AUXHDRS) elf_repl.h"; for file in $$files; do \
 	  if test -r $$file; then \
-	    $(INSTALL_DATA) $$file $(includedir)/libelf; \
+	    $(INSTALL_DATA) $$file $(DESTDIR)$(includedir)/libelf; \
 	  else \
-	    $(INSTALL_DATA) $(srcdir)/$$file $(includedir)/libelf; \
+	    $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(includedir)/libelf; \
 	  fi; \
 	done
 
@@ -185,31 +182,33 @@
 	uninstall-shared-$(DO_SHLIB) uninstall-compat-$(DO_COMPAT)
 
 uninstall-data:
-	$(RM) $(libdir)/libelf.a
-	$(RM) -r $(includedir)/libelf
+	$(RM) $(DESTDIR)$(libdir)/libelf.a
+	$(RM) -r $(DESTDIR)$(includedir)/libelf
 
 install-shared-yes: install-shared
 install-shared-no:
 install-shared: $(SHLIB)
-	$(INSTALL_SHLIB) $(SHLIB) $(libdir)
+	$(INSTALL) -d $(DESTDIR)$(libdir)
+	$(INSTALL_SHLIB) $(SHLIB) $(DESTDIR)$(libdir)
 	if test "$(SONAME)" = "$(SHLIB)"; then true; else \
-	  cd $(libdir) && $(RM) $(SONAME) && $(LN_S) $(SHLIB) $(SONAME); \
+	  cd $(DESTDIR)$(libdir) && $(RM) $(SONAME) && $(LN_S) $(SHLIB) $(SONAME); \
 	fi
 	if test "$(SHLINK)" = "$(SHLIB)"; then true; else \
-	  cd $(libdir) && $(RM) $(SHLINK) && $(LN_S) $(SHLIB) $(SHLINK); \
+	  cd $(DESTDIR)$(libdir) && $(RM) $(SHLINK) && $(LN_S) $(SHLIB) $(SHLINK); \
 	fi
 
 uninstall-shared-yes: uninstall-shared
 uninstall-shared-no:
 uninstall-shared:
-	cd $(libdir) && $(RM) $(SHLIB) $(SONAME) $(SHLINK)
+	cd $(DESTDIR)$(libdir) && $(RM) $(SHLIB) $(SONAME) $(SHLINK)
 
 install-compat-yes: install-compat
 install-compat-no:
 install-compat:
+	$(INSTALL) -d $(DESTDIR)$(includedir)
 	files="$(HDRS)"; for file in $$files; do \
-	  if test -f $(includedir)/$$file; then true; else \
-	    echo "#include <libelf/$$file>" > $(includedir)/$$file; \
+	  if test -f $(DESTDIR)$(includedir)/$$file; then true; else \
+	    echo "#include <libelf/$$file>" > $(DESTDIR)$(includedir)/$$file; \
 	  fi; \
 	done
 
@@ -217,8 +216,8 @@
 uninstall-compat-no:
 uninstall-compat:
 	files="$(HDRS)"; for file in $$files; do \
-	  if grep "^#include <libelf/$$file>$" $(includedir)/$$file >/dev/null 2>&1; then \
-	    $(RM) $(includedir)/$$file; \
+	  if grep "^#include <libelf/$$file>$" $(DESTDIR)$(includedir)/$$file >/dev/null 2>&1; then \
+	    $(RM) $(DESTDIR)$(includedir)/$$file; \
 	  else true; fi; \
 	done
 
--- libelf-0.8.2/po/Makefile.in~	Fri Jul 26 01:29:01 2002
+++ libelf-0.8.2/po/Makefile.in	Fri Jul 26 01:27:42 2002
@@ -91,18 +91,18 @@
 	catalogs="$(CATALOGS)"; for cat in $$catalogs; do \
 	  lang=`echo $$cat | sed 's,$(CATOBJEXT)$$,,'`; \
 	  dir=$(localedir)/$$lang/LC_MESSAGES; \
-	  $(SHELL) $(top_srcdir)/mkinstalldirs $$dir; \
+	  $(SHELL) $(top_srcdir)/mkinstalldirs $(DESTDIR)$$dir; \
 	  if test -r $$cat; then \
-	    $(INSTALL_DATA) $$cat $$dir/$(PACKAGE)$(INSTOBJEXT); \
+	    $(INSTALL_DATA) $$cat $(DESTDIR)$$dir/$(PACKAGE)$(INSTOBJEXT); \
 	  else \
-	    $(INSTALL_DATA) $(srcdir)/$$cat $$dir/$(PACKAGE)$(INSTOBJEXT); \
+	    $(INSTALL_DATA) $(srcdir)/$$cat $(DESTDIR)$$dir/$(PACKAGE)$(INSTOBJEXT); \
 	  fi; \
 	done
 
 uninstall:
 	catalogs="$(CATALOGS)"; for cat in $$catalogs; do \
 	  lang=`echo $$cat | sed 's,$(CATOBJEXT)$$,,'`; \
-	  $(RM) $(localedir)/$$lang/LC_MESSAGES/$(PACKAGE)$(INSTOBJEXT); \
+	  $(RM) $(DESTDIR)$(localedir)/$$lang/LC_MESSAGES/$(PACKAGE)$(INSTOBJEXT); \
 	done
 
 mostlyclean:
